src/App.jsx

```diff
@@ -0,0 +1,11 @@
+import React from "react"
+
+function App() {
+  return (
+    <div>
+      Dream Time News
+    </div>
+  );
+}
+
+export default App;
```

Comment:
Please use the arrow function version from start. It's a better practice. Also, make use of fragments rather than `<div>`

Reply:
done. app.jsx is now an arrow function and i changed the div